tabpy-server/server_tests/test_service_info_handler.py

```diff
@@ -0,0 +1,87 @@
+from tornado.testing import AsyncHTTPTestCase
+from tabpy_server.app import TabPyApp
+import simplejson as json
+from tabpy_server import __version__
+import tempfile, os
+
+
+def _create_expected_info_response(settings, tabpy_state):
+    response = {}
+    response['description'] = tabpy_state.get_description()
+    response['creation_time'] = tabpy_state.creation_time
+    response['state_path'] = settings['state_file_path']
+    response['server_version'] = settings['server_version']
+    response['name'] = tabpy_state.name
+    response['versions'] = settings['versions']
+    return response
+
+
+def _get_app_and_state(config_file=None):
+    tabpy_app = TabPyApp(config_file=config_file)
+    tornado_app = tabpy_app._create_tornado_web_app()
+
+    return tornado_app, tabpy_app.settings, tabpy_app.tabpy_state
+
+
+class TestServiceInfoHandlerDefault(AsyncHTTPTestCase):
+    def get_app(self):
+        app, self.settings, self.state = _get_app_and_state()
+        return app
+
+    def test_info(self):
+        """
+        Tests that /info returns the correct response on a vanilla TabPy instance.
```

Comment:
You can make test name something like `test_given_vanilla_tabpy_server_expect_correct_info_response` instead of adding docstring to a method :)